circuit/algorithms/build.rs

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2019-2022 Aleo Systems Inc.
+// This file is part of the snarkVM library.
+
+// The snarkVM library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// The snarkVM library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with the snarkVM library. If not, see <https://www.gnu.org/licenses/>.
+
+fn main() {
+    if cfg!(feature = "enable_console") {
```

Comment:
Why not just use the `enable_console` feature, instead of cfg(console)? It’s generally better to avoid build.rs if we can

Reply:
Good to know. What's the reason it's better?

Reply:
Sometimes build.rs can interfere with incremental compilation, which slows down iteration times

Reply:
I'll address in a future PR